test/UnitTests/MSTest.Analyzers.UnitTests/PreferConstructorOverTestInitializeAnalyzerTests.cs

```diff
@@ -83,7 +94,148 @@ public MyTestClass()
                 }
             }
             """;
+        string fixedCode = """
+            using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+            [TestClass]
+            public class MyTestClass
+            {
+                public MyTestClass()
+                {
+                }
+            }
+            """
+        ;
+
+        await VerifyCS.VerifyCodeFixAsync(code, fixedCode);
+    }
+
+    public async Task WhenTestClassHasTestInitializeAndCtorWithBody_Diagnostic()
+    {
+        string code = """
+            using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+            [TestClass]
+            public class MyTestClass
+            {
+                int x, y;
+                public MyTestClass()
+                {
+                    y=1;
+                }
+
+                [TestInitialize]
+                public void [|MyTestInit|]()
+                {
+                    x=1;
+                }
+            }
+            """;
+        string fixedCode = """
+            using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+            [TestClass]
+            public class MyTestClass
+            {
+                int x, y;
+                public MyTestClass()
+                {
+                    y=1;
+                    x=1;
+                }
+            }
+            """
+        ;
+
+        await VerifyCS.VerifyCodeFixAsync(code, fixedCode);
+    }
+
+    public async Task WhenTestClassHasTestInitializeAndCtorWithBothHavingBody_Diagnostic()
```

Comment:
This test is exactly the same as the previous one. I think it'd be good to have some block (like a `if`) and not simple statements.